### PR TITLE
Deployed config

### PR DESCRIPTION
**Pull request by eb16**
# :wrench: Bugfixes

# :recycle: Refactor

#### :heavy_check_mark:Tasks:

- Adds `try/catch` and dispatches error in every place we do an `async/await` call, so the app doesn't crash.
- Adds `static.json` with configuration for deployment.
- Changes stylelint config in `package.json` pointing to `.less` files instead of previous `.scss`.

---

#### :pushpin: Notes:

- There will be a PR with specific error handling.

---

#### :warning: Warnings:

- Default auth token only works in `develop` environment.

### CODE REVIEW

File: src/context/chat/actions.js [hunk 1]
```diff
@@ -1,12 +1,18 @@
 import ActionCable from 'actioncable'
 
 export const Actions = {
+  CHAT_ERROR: 'CHAT_ERROR',
```
Comment (AndresVidalBerriel) [inline src/context/chat/actions.js]: How about changing this action name (and the dispatcher method as well) to `SET_CHAT_ERROR`, using a verb to express the action, so it matches the naming of the other actions?

File: src/context/messages/actions.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 export const Actions = {
+  MESSAGES_ERROR: 'MESSAGES_ERROR',
```
Comment (AndresVidalBerriel) [inline src/context/messages/actions.js]: Same suggestion mentioned about `CHAT_ERROR` action. Could be changed to `SET_MESSAGES_ERROR`.

Comment (eb16) [reply src/context/messages/actions.js]: Done!

File: src/context/chat/actions.js (hunk 1 above)
Comment (eb16) [reply src/context/chat/actions.js]: I like consitency! Nice catch 💪

**Comment by AndresVidalBerriel** [comment]
Nice work, thank you for making the corrections!

**Pull request merged**